services/slam/builtin/cartographer_int_test.go

```diff
@@ -156,8 +176,136 @@ func TestCartographerIntegration(t *testing.T) {
 
 	// Close out slam service
 	test.That(t, utils.TryClose(context.Background(), svc), test.ShouldBeNil)
+
```

Comment:
[nit/opt] You could split off an integration test helper function like in orbslam_int_test for future proofing testing when we are also testing taking in 3d data into cartographer. Or you can wait for that to happen before you split off the helper, up to you